roles/public_facing/files/download.ceph.com/rrsync

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/perl
```

Comment:
I'm waiting to hear back from the bitergia folks if we can just use the version that comes with the rsync package.